tls/s2n_resume.c

```diff
@@ -627,21 +650,20 @@ int s2n_encrypt_session_ticket(struct s2n_connection *conn, struct s2n_ticket_fi
     POSIX_GUARD(s2n_stuffer_write_bytes(&aad, key->implicit_aad, S2N_TICKET_AAD_IMPLICIT_LEN));
     POSIX_GUARD(s2n_stuffer_write_bytes(&aad, key->key_name, S2N_TICKET_KEY_NAME_LEN));
 
-    struct s2n_blob state_blob = { 0 };
-    struct s2n_stuffer state = { 0 };
+    uint32_t unencrypted_size = s2n_stuffer_data_available(to);
```

Comment:
`unencrypted_size` is the length of the plaintext header? That is actually pretty confusing and took me far too many reads to get. Maybe name it `plaintext_header_size` or `plaintext_prefix_size` or similar.